blockchain/blockchain.go

```diff
@@ -848,3 +851,7 @@ func (bc *blockchain) replaceHeightAndHash(blk *Block) (uint64, hash.Hash32B, er
 	}
 	return tipHeight, tipHash, nil
 }
+
+func (bc *blockchain) SubscribeToBlock(ch chan *Block) error {
+	return bc.dao.AddSubscriber(ch)
```

Comment:
The subscriber doesn't need to be push down to dao. It doesn't seem to use any dao level context.